sub-projects/device/src/AWSClass.py

```diff
@@ -0,0 +1,45 @@
+import boto3
+import json
+
+#this class is for telescreen_device activity
```

Comment:
In line comments should only be above tricky parts of the code.

> Complicated operations get a few lines of comments before the operations commence. Non-obvious ones get comments at the end of the line.

I will only comment on this occurrence, but there are many other occurrences below.
see https://google.github.io/styleguide/pyguide.html#Comments